src/cdk/drag-drop/drag-drop.md

```diff
@@ -128,6 +128,17 @@ to be applied.
 
 <!-- example(cdk-drag-drop-custom-preview) -->
 
+### Customizing the drag preview container
+When a `cdkDrag` element is picked up, it will create a preview element which is appended to
+the end of `<body>`. There may be times when you need to have the preview created elsewhere.
+
+This can be configured using the `cdkDragPreviewContainer` attribute.
+Passing `global` will respect the default behavior and the preview will be appended to `<body>`.
+Passing `parent` will append the preview to the parent of `cdkDropList`.
+You can also pass in `TemplateRef` or `ElementRef` for finer control.

```

Comment:
I don't think a `TemplateRef` makes sense since templates aren't inside the DOM.

Reply:
Yeah I was using the wrong term, there is not enough coffee in the world!
I've removed `TemplateRef`